src/email.rs

```diff
@@ -79,32 +94,37 @@ impl Config {
             if let Some(attachment) = &self.attachment {
                 let filename = attachment
                     .file_name()
-                    .ok_or_else(|| anyhow::anyhow!("Failed to get filename"))?
+                    .ok_or_else(|| EmailError::ReadError)?
                     .to_string_lossy();
-                let filebody = fs::read(attachment)?;
+                let filebody = fs::read(attachment).map_err(|_| EmailError::ReadError)?;
                 let content_type = ContentType::parse(
                     mime_guess::from_path(attachment)
                         .first_or_octet_stream()
                         .as_ref(),
-                )?;
+                )
+                .map_err(|_| EmailError::ReadError)?;
```

Comment:
returns a [`lettre::message::header::ContentTypeErr`](https://docs.rs/lettre/latest/lettre/message/header/struct.ContentTypeErr.html), use it